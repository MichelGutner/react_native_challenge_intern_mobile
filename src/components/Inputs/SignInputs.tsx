import React from 'react';
import styled from 'styled-components/native';

const Input = styled.TextInput`
  margin-top: 16px;
  width: 312px;
  height: 48px;
  background-color: #0050f0;
  border: 1px solid #ffffff;
  border-radius: 4px;
  font-family: Roboto;
  font-style: normal;
  font-weight: normal;
  font-size: 16px;
  display: flex;
  align-items: center;
  text-align: center;
`;

const SignInputs = ({ ...rest }) => {
  return <Input {...rest}></Input>;
};

export default SignInputs;
