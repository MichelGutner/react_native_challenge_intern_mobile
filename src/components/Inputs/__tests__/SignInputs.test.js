import React from 'react';
import renderer from 'react-test-renderer';
import SignInputs from '../SignInputs';

test('renders correctly', () => {
  const tree = renderer.create(<SignInputs />).toJSON();
  expect(tree).toMatchSnapshot();
});
