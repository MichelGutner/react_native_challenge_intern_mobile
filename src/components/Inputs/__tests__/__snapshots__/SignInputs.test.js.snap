// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<TextInput
  style={
    Array [
      Object {
        "alignItems": "center",
        "borderRadius": 4,
        "color": "white",
        "display": "flex",
        "fontFamily": "Roboto",
        "fontSize": 16,
        "fontStyle": "normal",
        "fontWeight": "normal",
        "height": 48,
        "textAlign": "center",
        "width": 290,
      },
    ]
  }
/>
`;
